lib/graphql/errors.rb

```diff
@@ -44,7 +44,13 @@ def rescue_from(*classes, &block)
       raise EmptyRescueError unless block
 
       classes.each do |klass|
-        raise NotRescuableError.new(klass.inspect) unless klass.is_a?(Class)
+        key = if klass.is_a?(Module) && klass.respond_to?(:===)
+          klass.name
+        elsif klass.is_a?(String)
```

Comment:
This point is arguable, I just took the solution from Rails, the first part 
```ruby
if klass.is_a?(Module) && klass.respond_to?(:===)
```
 is enough for my own case, it'll handle both Class and Module as an argument, but...
 it's up to you, whether it reasonable to use String as a klass or not